### PR TITLE
Security Officer, Detective and Warden spawn with tablets.

### PR DESCRIPTION
**Pull request by GuillaumePrata**
## About The Pull Request
It's a goal to yeet PDAs into the sun and move to modular consoles so this change is aimed to be a step on that direction.
The security tablets are given to Officers, Detective and the Warden.
They come pre installed with BotKeeper and the Crew manifest viewer, both functions that are build in on their PDA cartridges already.

## Why It's Good For The Game
Security was the department without any roundstart tablet (ignoring that the HoS starts with the generic Command one) and while the programs that we have right now are still lacking in function to their PDA cartridge counterparts like the security records, we already have a few that will help them enjoy the new UI from the tablets.

## Changelog

:cl: Guillaume Prata
expansion: Security Officers, Detectives and the Warden will now spawn with a Tablet pre installed with BotKeeper and Crew Manifest.
/:cl:

### CODE REVIEW

**Comment by JohnFulpWillard** [comment]
maybe now we can add a security records app (with view-only mode so lawyers can also use it)

**Comment by wesoda25** [comment]
Pdas are better

**Comment by CRITAWAKETS** [comment]
i think the warden should actually get a laptop, both to reinforce to new players the idea that laptops like, exist, but to also get the security camera app to be more prominent. Also would show that modular computers are not too different in console and portable item form, showcasing how they can be used.

**Comment by zxaber** [comment]
To be clear, the goal (or maybe it's just *my* goal) is to make a new class of modPC smaller than tablets and give everyone that. I've taken to calling it "porting PDAs to ModPC" rather than "replacing PDAs". Tablets will generally go to heads of staff for roundstart equipment.

That said, this doesn't hurt anything in the meantime. Ideally, most nuPDAs will have the same job-specific functions that old PDAs have.

File: code/modules/modular_computers/computers/item/tablet_presets.dm [hunk 1]
```diff
@@ -65,6 +65,12 @@
 	hard_drive.store_file(new /datum/computer_file/program/alarm_monitor)
 	hard_drive.store_file(new /datum/computer_file/program/supermatter_monitor)
 
+/obj/item/modular_computer/tablet/preset/advanced/security/Initialize(mapload)
+	. = ..()
+	var/obj/item/computer_hardware/hard_drive/small/hard_drive = find_hardware_by_name("solid state drive")
```
Comment (ShizCalev) [inline code/modules/modular_computers/computers/item/tablet_presets.dm:70]: jesus fuck i hate this why isn't it a define at the least

**Pull request merged**